src/sysupdate/sysupdate-partition.c

```diff
@@ -27,7 +27,7 @@ int read_partition_info(
         _cleanup_free_ char *label_copy = NULL, *device = NULL;
         const char *label;
         struct fdisk_partition *p;
-        uint64_t start, size, flags;
+        uint64_t start, size, flags, sector_size;
```

Comment:
Can you rename `sector_size` to `ssz`, that's what we use everywhere else.

Reply:
Also, can you make it of type `unsigned long` as that is what is returned by `fdisk_get_sector_size()`?

Reply:
Updated as requested.